src/py_eddy_tracker/dataset/grid.py

```diff
@@ -1275,7 +1275,7 @@ def get_step_in_km(self, lat, wave_length):
         min_wave_length = max(step_x_km, step_y_km) * 2
         if wave_length < min_wave_length:
             logger.error(
-                "Wave_length too short for resolution, must be > %d km",
+                "Wavelength too short for resolution, must be > %d km",
```

Comment:
In this case we talked about variable which are called "wave_length", i don't know the best soltuion